tests/numeric_test.py

```diff
@@ -1394,3 +1619,156 @@ def testPrecision(self):
         _, floatmean = g.mean(floatval)
         ak_mse = ak.mean((intmean - floatmean) ** 2)
         self.assertTrue(np.isclose(ak_mse, 0.0))
+
+    # test clip on ints, floats, and mash-ups; note if any input is float, output is float
+
+    def testClip(self):
```

Comment:
make sure to also add this to `PROTO_test`. And we tend to use `_` instead of camelCase on the python side so `test_clip`

Reply:
amanda already pointed this out!